litmus-portal/backend/auth/pkg/manage/manager.go

```diff
@@ -72,20 +86,40 @@ func (m *Manager) VerifyUserPassword(username, password string) (*models.PublicU
 	return user.GetPublicInfo(), err
 }
 
+// LoginUser verifies user password
+func (m *Manager) LoginUser(username string) error {
+	storedUser, err := m.GetUser(username)
+	if err != nil {
+		return err
+	}
+	storedUser.LoggedIn = true
+	return m.userStore.UpdateUser(storedUser)
+}
+
+// LogoutUser verifies user password
+func (m *Manager) LogoutUser(username string) error {
```

Comment:
This logic won't work, if the user closes the browser directly without logging out. This would not be triggered. If we are planning to track the logins this method is not optimal.

Reply:
We can extend this  logout functionality with the help of jwt. But we need to have this endpoint in order to update the database about the login info. We can surely extend it in the next releases